### PR TITLE
[6.5] Secure element extension

### PR DESCRIPTION
**Pull request by pkosko**
[ACR] https://code.sec.samsung.net/jira/browse/TWDAPI-281

### CODE REVIEW

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html [hunk 1]
```diff
@@ -1514,6 +1560,14 @@ <h2 id="api-features">2. Related Feature</h2>
            </div></p>
 <li class="feature">http://tizen.org/feature/network.secure_element</li>
 </div>
+<div class="def-api-feature">
+<p><div class="description">
+            <p>
+To guarantee this application will run on a device which supports the USB secure element feature, add the below feature declaration to the config file:
```
Comment (annie-samsung) [inline docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1566]: To guarantee? 
Can we reword to something like
To ensure this application can run on a device...

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html [hunk 2]
```diff
@@ -1322,6 +1326,48 @@ <h4>Methods</h4>
...
+<div class="synopsis"><pre class="signature prettyprint">void onUSBCardInserted(<a href="#Reader">Reader</a> reader);</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">reader</span>:
+ An inserted USB Secure Element reader.
```
Comment (annie-samsung) [inline docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1345]: Is this sentence complete?

USB Secure Element reader is inserted.

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html [hunk 3]
```diff
@@ -1322,6 +1326,48 @@ <h4>Methods</h4>
...
+<div class="synopsis"><pre class="signature prettyprint">void onUSBCardRemoved(<a href="#Reader">Reader</a> reader);</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">reader</span>:
+ A removed USB Secure Element reader.
```
Comment (annie-samsung) [inline docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1366]: USB Secure Element reader is removed.

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html (hunk 1 above)
Comment (pwasowski2) [reply docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1566]: We use this exact wording "To guarantee this application will run..." in several API references (or even all of them) - we used it here to be consistent with other docs.

Do you think, we should change it?

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html (hunk 2 above)
Comment (pwasowski2) [reply docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1345]: Yes, this sentence is correct - `reader` is an object, that represents the USB Secure Element reader that is inserted.

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html (hunk 3 above)
Comment (pwasowski2) [reply docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1366]: This sentence is correct - `reader` is an object, that represents the USB Secure Element reader that is inserted.

File: docs/application/web/api/6.5/device_api/mobile/tizen/se.html (hunk 1 above)
Comment (annie-samsung) [reply docs/application/web/api/6.5/device_api/mobile/tizen/se.html:1566]: If this has been used consistently and you want to use the same let's keep it that way.

**Review by annie-samsung — APPROVED** [review]
Approving

**Pull request merged**